### PR TITLE
Fix download link disappearing

### LINKED ISSUE #210: Download link disappears in certain resolutions
The download link disappears on the homepage when the width of the browser window is between about this range (apologies for my poor CSS/HTML debugging skills): `1201-1284px`

Happens with the latest versions of Chrome 74 and Firefox 67.

Download link is visible:
Screen Shot 2019-05-27 at 9 00 22 PM
It disappears:
Screen Shot 2019-05-27 at 9 00 33 PM
Link is visible again once on a mobile resolution:
Screen Shot 2019-05-27 at 9 00 42 PM
Does disappear again though (probably not a big deal for a mobile user though):
Screen Shot 2019-05-27 at 9 05 40 PM

### PR DESCRIPTION
**Pull request by taylorsilva**
Fixes #210

Only tested on Chrome 74 and Firefox 67. I was able to build and render the site without any other visible issues.

### CODE REVIEW

File: css/booklit.css [hunk 1]
```diff
@@ -743,6 +743,7 @@ h3 .anchor {
 }
 .top-nav {
   position: relative;
+  white-space: nowrap;
```
Comment (vito) [inline css/booklit.css]: This change needs to be made to `less/booklit.less` instead and then compiled :slightly_smiling_face:

Comment (taylorsilva) [reply css/booklit.css]: Knew I'd miss something. I don't understand this newer css stuff :(

Comment (taylorsilva) [reply css/booklit.css]: I initially misunderstood your comment. Thought I had to _only_ make the change in the `/less` directory, but I had to make the change in both `less` and `css`. I needlessly deleted my commit haha.

**Comment by taylorsilva** [comment]
I initially misunderstood your comment. Thought I had to _only_ make the change in the `/less` directory, but I had to make the change in both `less` and `css`. I needlessly deleted my commit haha.

**Pull request merged**